meta-refkit-core/recipes-ostree/refkit-ostree/files/refkit-ostree/src/refkit-ostree.c

```diff
@@ -0,0 +1,1642 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
```

Comment:
I have not reviewed the new C code in detail. Has anyone else reviewed it before and was static code analysis done for it?